src/lib.rs

```diff
@@ -293,4 +515,188 @@ mod test {
 
         assert_eq!(env.compile(), expected);
     }
+
+    #[test]
+    fn var_init() {
+        let v: Var = ("KEY", "VALUE").into();
+        let v2 = Var {
+            environment: Environment::empty(),
+            condition: true,
+            key: "KEY".into(),
+            value: "VALUE".into(),
+        };
+
+        assert_eq!(v, v2);
+    }
+
+    #[test]
+    fn environment_var() {
+        let v: Var = ("FOO", "BAR").into();
+
+        ::std::env::set_var("FOO", "BAR");
+
+        let e = Environment::inherit();
```

Comment:
Ideally the tests are able to run in parallel.

1. This would mean avoid relying on `set_var` unless the test requires it
2. Using different keys in different tests so they don't conflict

(this is what got me to contributing to assert_cli, I needed my tests to run in different directories but setting the tests cwd ran into race conditions)